### PR TITLE
Datepicker for Expiration

### PR DESCRIPTION
**Pull request by Vivek205**
Fix for issue - #192 
- Set Date picker for selecting the expiry date
- Default date will be populated based on the threshold block number
- Block number is calculated and displayed based on the date selected
- Block number will be always disabled
- Dates less than the min exp date are disabled.

### CODE REVIEW

File: src/components/JobDetails.js [hunk 1]
```diff
@@ -93,6 +105,20 @@ export  class Jobdetails extends React.Component {
       return this.channelHelper.reInitialize(channelInfoUrl);
     }
 
+    initExpBlockDate(setMinExpDate=false){
+      let expBlockNumber =  this.serviceState['payment_expiration_threshold']+BLOCK_OFFSET;
+      let expDays = Math.ceil(expBlockNumber *15/(24 * 3600));
```
Comment (raamb) [inline src/components/JobDetails.js]: Lets make 15 a constant in the utils class. Call it BLOCK_TIME_SECONDS

Comment (Vivek205) [reply src/components/JobDetails.js]: BLOCK_TIME_SECONDS is stored in util class.

**Comment by Vivek205** [comment]
Updated the missing import. Please review now.

**Comment by Vivek205** [comment]
Included styles for date picker ( both active and disabled state)

**Comment by Vivek205** [comment]
Changed the condition for comparing the expiration provided and the threshold

**Pull request merged**